Workbooks/Forcepoint Cloud Secuirty Gateway workbook.json

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
No space in file name please.
Rename the workbook file name to ForcepointCloudSecuirtyGatewayworkbook.json